### PR TITLE
NPM Install Cleanup

### PR DESCRIPTION
**Pull request by frytyler**
When I forked the repo to start working on #49 I noticed that there was a `yarn.lock` file. Trying to run `yarn install` threw a bunch of errors but `npm install` worked correctly. Given both approaches seemed to be in this package I am suggesting `yarn` is remove in favor of npm. 

While cleaning out `yarn` I fixed up a couple other warnings given npm updates and elint. 

This will help when people jump in to contribute.

### CODE REVIEW

**Comment by tizmagik** [comment]
Thanks for the PR!

File: package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     "react-test-renderer": "^15.5.4"
   },
   "scripts": {
-    "prepublish": "npm run lint && npm run test && npm run build",
+    "prepare": "npm run lint && npm run test && npm run build",
```
Comment (tizmagik) [inline package.json]: Is prepublish deprecated? Why this change, just out of curiosity ?

Comment (frytyler) [reply package.json]: Ya sorry, `As of npm@5, `prepublish` scripts are deprecated.` was the warning. Here's more on that. https://docs.npmjs.com/misc/scripts#prepublish-and-prepare

**Comment by frytyler** [comment]
@tizmagik There is already a `package-lock` file. I'm pushing the travis file update now.

File: package.json (hunk 1 above)
Comment (frytyler) [reply package.json]: It's worth noting this was just a warning and could be done in a future PR when it's more of an error. `prepare` seems to have been around in npm a while so there should be a concern of that hook not being available.

Comment (tizmagik) [reply package.json]: Ah that's right. This is good then, thanks! 👍

File: .travis.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 language: node_js
 node_js:
   - 6
-script: yarn && yarn run lint && yarn test
+script: npm run lint && npm test
```
Comment (tizmagik) [inline .travis.yml]: Can this also `npm run build` -- I want to guard against any possible issues creating a build

**Comment by frytyler** [comment]
@tizmagik I just noticed that build was at the end in the package file was about to change it. but nvm 👍

**Comment by tizmagik** [comment]
This is great, thanks so much @frytyler ! 🎉
